### ISSUE TITLE
Sankey diagram doesn't fire node click event

### ISSUE THREAD

**Issue by jarben** [ecomfe/echarts#3188]
I've noticed that sankey nodes don't fire any mouse events if I do: 

chart.on("click", function(){
    console.log(arguments); // nothing is logged
});

Would be great if this is possible, this would be helpful if some links are highlighted on node click such as: 

![image](https://cloud.githubusercontent.com/assets/561365/15185157/3af54afc-1798-11e6-8f42-530d550a2346.png)

**Comment by chaegumi** [comment]
http://echarts.baidu.com/demo.html#sankey-energy

I run this code.It seem work well.

```javascript

myChart.showLoading();
$.get('data/asset/data/energy.json', function (data) {
    myChart.hideLoading();

    myChart.setOption(option = {
        title: {
            text: 'Sankey Diagram'
        },
        tooltip: {
            trigger: 'item',
            triggerOn: 'mousemove'
        },
        series: [
            {
                type: 'sankey',
                layout: 'none',
                data: data.nodes,
                links: data.links,
                itemStyle: {
                    normal: {
                        borderWidth: 1,
                        borderColor: '#aaa'
                    }
                },
                lineStyle: {
                    normal: {
                        curveness: 0.5
                    }
                }
            }
        ]
    });
});
myChart.on('click', function(params){
    console.log(arguments);
});

```

**Comment by jarben** [comment]
Looks like it's now fixed in v3.1.9, but didn't work in v3.1.7

Anyway, thanks for looking at this, this sankey diagram is awesome!
